### ISSUE TITLE
General error: 1 no such table: permissions (Connection: sqlite, SQL: select * from "permissions")

### ISSUE THREAD

**Issue by Houbsi** [eramitgupta/laravel-role-permission#15]
I needed to refresh my database, like always during development. And now i get the following error. Even when i try to do a composer update. (Before this happened i got it updated to the latest version from today.

i

```bash
php artisan migrate

   Illuminate\Database\QueryException 

  SQLSTATE[HY000]: General error: 1 no such table: permissions (Connection: sqlite, SQL: select * from "permissions")

  at vendor\laravel\framework\src\Illuminate\Database\Connection.php:825
    821▕                     $this->getName(), $query, $this->prepareBindings($bindings), $e
    822▕                 );
    823▕             }
    824▕ 
  ➜ 825▕             throw new QueryException(
    826▕                 $this->getName(), $query, $this->prepareBindings($bindings), $e
    827▕             );
    828▕         }
    829▕     }

  1   [internal]:0
      Illuminate\Foundation\Application::Illuminate\Foundation\{closure}(Object(EragPermission\PermissionServiceProvider), "EragPermission\PermissionServiceProvider")

  2   vendor\laravel\framework\src\Illuminate\Database\Connection.php:407
      PDOException::("SQLSTATE[HY000]: General error: 1 no such table: permissions")
```

It shows me that it fails here:

```php
    Permission::with('roles.users')->get()->each(function ($permission) {
            Gate::define($permission->name, function ($user) use ($permission) {
                return $user->hasPermissionTo($permission);
            });
        });
``` 

For troublleshooting i removed everything (package, view, routes,...) But i'm still getting the error that it's searching for the permissions table...

**Comment by Houbsi** [comment]
`php artisaan migrate` shows the same.

**Comment by eramitgupta** [comment]
I just noticed I fixed it
@Houbsi Thanks 🤌

**Comment by Houbsi** [comment]
Okay, i wasn't going crazy then :D 

I somehow got it working with SESSION_DRIVER set to file and not database anymore.

**Comment by eramitgupta** [comment]
Fixed Database Connection

**Comment by Houbsi** [comment]
Can confirm. reverted back, it works with the new version. old version throws error. Thanks for the quick fix @eramitgupta !

**Comment by Houbsi** [comment]
Fixed with d2bc686286f53ed1adfadae4797503682b83ae77
